velox/functions/prestosql/aggregates/ApproxMostFrequentAggregate.cpp

```diff
@@ -32,6 +32,10 @@ struct ApproxMostFrequentAggregate : exec::Aggregate {
     return sizeof(StreamSummary);
   }
 
+  bool accumulatorUsesExternalMemory() const override {
```

Comment:
Why does it use external memory? I expect it to use HashStringAllocator.

Reply:
OK I will remove these.  So if it's from `HashStringAllocator`, we don't need to call the destructor on accumulator since we will release the memory pool when the task is done?

Reply:
If we are clearing memory in bulk, then we don't need to clear individual accumulators. RowContainer::freeAggregates does call Aggregate::destroy to clear some, but not all accumulators.